packages/wonder-blocks-core/components/clickable-behavior.js

```diff
@@ -288,16 +283,23 @@ export default class ClickableBehavior extends React.Component<Props, State> {
 
     handleTouchStart = () => {
         this.setState({pressed: true});
+        this.handlingTouch = true;
     };
 
-    handleTouchEnd = () => {
+    handleTouchEnd = (evt: SyntheticMouseEvent<*>) => {
+        evt.stopPropagation();
+        evt.preventDefault();
         this.setState({pressed: false});
-        this.waitingForClick = true;
+        this.handleClick(evt);
```

Comment:
I think this may mean that we end up calling `handleClick` twice on mobile, correct? It seems like both touchend/touchcancel and click are fired: https://www.html5rocks.com/en/mobile/touchandmouse/

I think your changed test may indicate that issue, as well (since the same number of events are firing even though a click event hasn't been triggered).

Reply:
see https://developer.mozilla.org/en-US/docs/Web/API/Touch_events/Supporting_both_TouchEvent_and_MouseEvent#Event_order

> If the touchstart, touchmove or touchend event is canceled during an interaction, no mouse or click events will be fired, and the resulting sequence of events would just be:
> 
> - touchstart
> - Zero or more touchmove events, depending on movement of the finger(s)
> - touchend